qubesmanager/tests/test_qube_manager.py

```diff
@@ -1604,3 +1604,55 @@ def test_704_check_later(mock_timer, mock_question):
 
     assert mock_question.call_count == 0
     assert mock_timer.call_count == 1
+
+
+@pytest.mark.asyncio(loop_scope="module")
+async def test_705_prohibit_start_vms(qubes_manager):
```

Comment:
This tests misses the actual "test that something happened" part. In particular, after selecting a VM you are not checking if, for example, the action menu options are appropriate.